tests/Feature/Store/SearchPageTest.php

```diff
@@ -263,10 +263,11 @@ public function itPaginatesWhenRequired()
         ]);
 
         // Visit search page, make sure next page link is present and works
+        // <a class="page-link" href="http://arcv-store.test/registrations?page=2" rel="next" aria-label="Next »">›</a>
         $this->actingAs($centreUser, 'store')
             ->visit(URL::route('store.registration.index'))
-            ->see('<a href="' . URL::route('store.base') . '/registrations?page=2' . '" rel="next">»</a>')
-            ->click('»')
+            ->see('<a class="page-link" href="' . URL::route('store.base') . '/registrations?page=2' . '" rel="next" aria-label="Next »">›</a>')
```

Comment:
wondering if  `aria-label="Next »"` needs the `»`

Reply:
It seems unhelpful that it's there. I guess this is the default that Illuminate\Pagination\LengthAwarePaginator is deciding should be there?